rpc/rpc.proto

```diff
@@ -23,18 +25,21 @@ message AddNetworkRequest {
 message AddNetworkReply {
   bool Success = 1;
   string IPv4Addr = 2;
+  string IPv6Addr = 3;
   int32 DeviceNumber = 4;
   bool UseExternalSNAT = 5;
-  repeated string VPCcidrs = 6;
+  repeated string VPCV4cidrs = 6;
+  repeated string VPCV6cidrs = 7;
+
 
   // start of pod-eni parameters
-  int32 PodVlanId = 7;
-  string PodENIMAC = 8;
-  string PodENISubnetGW = 9;
-  int32 ParentIfIndex = 10;
+  int32 PodVlanId = 8;
+  string PodENIMAC = 9;
+  string PodENISubnetGW = 10;
+  int32 ParentIfIndex = 11;
```

Comment:
Since, CNI is the only consumer for this and since we always release both CNI and IPAMD together in a release with these changes applied on both ends - I thought this should be alright. I mean we are never going to match this IPAMD version with an older CNI version - at least that was the thought process that led to structuring it in this fashion. I'll fix it and will instead append the new fields at the end.

Reply:
There's a version skew (briefly, and mostly only theoretically) between client and server during upgrade/downgrade.
Eg: on an overloaded system, an ADD/DEL might start the client binary, but then aws-node is replaced with a different version before the client binary proceeds.  We don't need that to _work_ as such (it's ok if it aborts and retries), but we do need it to not do something crazy like exit successfully but with a completely bogus IP because it's misparsed one of the existing protobuf fields as some other field.